test/spec/messageEvents.spec.js

```diff
@@ -35,15 +39,17 @@ describe('Message Events Library', function() {
         to: '2016-11-14T16:15',
         transmission_ids: '65832150921904138'
       };
-      messageEvents.search(options, function(err, data) {
-        Object.keys(options).forEach(function(key) {
-          expect(client.get.firstCall.args[0].qs).to.have.property(key).and.equal(options[key]);
+      messageEvents.search(options)
```

Comment:
you'll need to return this promise as well to tell mocha to wait for the test, otherwise it will move on before the test completes